ckanext/opendata_theme/base/helpers.py

```diff
@@ -121,6 +121,14 @@ def is_data_dict_active(ddict):
     return False
 
 
+def is_segment_active():
+    return config.get('ckan.segment_active', 'false').lower() == 'true'
```

Comment:
To parse boolean values it's better to use the following
```
toolkit.asbool(config.get(ckan.segment_active, False))
```

Reply:
Thanks for the feedback. I'll actually change the way this works. Instead of using 2 attributes (isActive and getWriteKey), I'll just use getWriteKey, and if the value is empty, that disables segment. It's simpler and cleaner